app/javascript/sidebar-widget/SidebarWidget.jsx

```diff
@@ -1,4 +1,5 @@
 import { h, Component } from 'preact';
+import Honeybadger from 'honeybadger-js';
```

Comment:
Thanks for making the change @frkozbk! Just one more little change. `HoneyBadger` is available globally so you do not need to import it. eslint will complain that it does not exist, so you can just add it to the globals section of https://github.com/thepracticaldev/dev.to/blob/master/app/javascript/.eslintrc.js